app/assets/javascripts/general.js

```diff
@@ -1,31 +1,46 @@
 $(document).ready(function() {
  // flash message for people coming from other countries
- if(window.location.search.substring(1).search("country_name") == -1) {
-    if (!$.cookie('has_seen_country_message')) {
-  $.ajax({
-    url: "/country_message",
-        dataType: 'html',
-        success: function(country_message){
+ var htmlWrapperFront = [
```

Comment:
my JavaScript is *very* rusty. This is likely bad code, feel free to rip it apart